types/test.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react'
-import {render, fireEvent, screen, waitFor} from '@testing-library/react'
-import * as pure from '@testing-library/react/pure'
+import {render, fireEvent, screen, waitFor} from '.'
+import * as pure from './pure'
```

Comment:
I couldn't get the `paths` config set to make this work like it was before. I'm personally not that bothered by this, but if someone cares, feel free to update it :)